Library/Formula/stack.rb

```diff
@@ -0,0 +1,15 @@
+class Stack < Formula
+  desc "A cross-platform program for developing Haskell projects."
+  homepage "https://github.com/commercialhaskell/stack"
+  url "https://github.com/fpco/stack/releases/download/v0.0.0-beta/stack-0.0.0-x86_64-osx.gz"
```

Comment:
Oh cool. Is that new? I thought it built everything from source! Is it *always* binaries that are distributed? Or does it try and then fall back to a local compilation?

I'm not sure when I'll have time to do this, that will be quite a bit more work. Will try and do it sometime, but someone else will likely beat me to it.